tests/test-definition.yaml

```diff
@@ -5,30 +5,28 @@ dimensions:
       - "false"
   - name: spark
     values:
-      - 3.5.5
       - 3.5.6
+      - 3.5.7
       - 4.0.0
```

Comment:
In the Google sheet (source of truth) we talk about Spark 4.0.1 and not 4.0.0.
I think we should either bump this PR to 4.0.1 or if too much effort decide to not ship 4.0.1 for now